workflow/exp/exp.rt_jet_conus12km

```diff
@@ -4,22 +4,26 @@
 export NET=rrfs                    # rrfs, rtma, 
 export MESH_NAME=conus12km         # conus12km, conus3km, atl3km, etc
 export WGF="det"   # working group function = "det", "enkf", "ensf", "firewx"
-export EXP_NAME=rrfs${WGF}
-export VERSION=v2.0.3
-export TAG=c3v203
-export OPSROOT=/lfs5/BMC/nrtrr/NCO_dirs/${VERSION}
-export EXPDIR=${OPSROOT}/${EXP_NAME}
+export EXP_NAME=dev12km
+export VERSION=v2.0.9
+export TAG=c3v209
+export OPSROOT=/lfs5/BMC/nrtrr/NCO_dirs
+export EXPDIR=${OPSROOT}/exp/${EXP_NAME}/rrfs${WGF}
 export COMROOT=${OPSROOT}/com      # task input and output data as well as logs
 export DATAROOT=${OPSROOT}/stmp    # task workdirs ($DATA) which to be removed immediately upon task completion unless KEEPDATA=yes
 #
 export REALTIME=true     # true, false
 #
-export PHYSICS_SUITE="convection_permitting" # mesoscale_reference, convection_permitting
+export PHYSICS_SUITE="convection_permitting"   # mesoscale_reference, convection_permitting
 export ZETA_LEVELS="L60.txt"
 #
 export DA_IODA="true"
 export DA_JEDI="true"
 export OBSPATH="/public/data/grids/rap/obs"
+export HYB_OPT=0  # 0: no_ensemble BEC; 1. online rrfs ensembles; 2. offline rrfs ensembles; 3. online GDAS ensembles
```

Comment:
What should it be for pure EnVar?

Reply:
Good question. 
I had thought we could set the static weight to 0 so that we could run pure EnVar.
I created a related issue at https://github.com/JCSDA-internal/saber/issues/1006